ddosa.py

```diff
@@ -349,7 +349,6 @@ def hash_to_path2(hashe):
 def store_renga(filepath,obj):
     import renga
     client = renga.from_env()
-    client.endpoint = "http://172.18.0.5"
```

Comment:
clearly you do not need this, thanks.
but it was a quick fix, needed because the endpoint from the env was not correct. I am not sure why, it took some time to figure out this was the problem. I suspect it had something to do with multiple different docker networks in my laptop, or some other local specifics. If I have an opportunity I will describe this in detail.